app/javascript/components/explore/CellFilteringPanel.jsx

```diff
@@ -31,6 +30,244 @@ export function CellFilteringPanelHeader({
 }
 
+/**
+ * Very brief notes on terms in SCP metadata convention, derived from:
+ * https://singlecell.zendesk.com/hc/en-us/articles/360060609852-Required-Metadata
+ */
+const conventionalMetadataGlossary = {
+  'biosample_id': 'Unique identifier for each sample in the study',
+  'donor_id': 'Unique identifier for each biosample donor in the study',
+  'species__ontology_label': 'Taxon name, from NCBITaxon',
+  'disease__ontology_label': 'Disease name, from Mondo or PATO',
+  'organ__ontology_label': 'Organ name, from Uberon',
+  'library_preparation_protocol__ontology_label': 'From EFO',
+  'sex': 'One of "female", "male", "mixed", or "unknown"',
+  'cell_type__ontology_label': 'From Cell Ontology',
+  'ethnicity__ontology_label': 'From Human Ancestry Ontology'
+}
+
+/** Determine if annotation is conventional from its raw name */
+function getIsConventionalAnnotation(rawName) {
+  return (
+    rawName.includes('__ontology_label') ||
+    ['donor_id', 'biosample_id'].includes(rawName)
+  )
+}
+
+/** Convert e.g. "cell_type__ontology_label" to "Cell type" */
+function parseAnnotationName(annotationIdentifier) {
+  const rawName = annotationIdentifier.split('--')[0]
+  const sansOntologyName = rawName.replace('__ontology_label', '')
+  const sentenceCased = sansOntologyName[0].toUpperCase() + sansOntologyName.slice(1)
+  const spaced = sentenceCased.replace(/_/g, ' ')
+  let upId = spaced
+  if (spaced.slice(-3) === ' id') {
+    // e.g. Donor id -> Donor ID
+    upId = upId.slice(0, -3) + upId.slice(-3).toUpperCase()
+  }
+  let ynLess = upId
+  if (upId.slice('-3').toUpperCase() === ' YN') {
+    ynLess = ynLess.slice(0, -3)
+  }
+  const displayName = ynLess
+  return [displayName, rawName]
+}
+
+/** Toggle icon for collapsing a list; for each filter list, and all filter lists */
+function CollapseToggleChevron({ isCollapsed, whatToToggle, isLoaded }) {
+  let toggleIcon
+  let toggleIconTooltipText
+  if (!isCollapsed) {
+    toggleIcon = <FontAwesomeIcon icon={faChevronDown} />
+    toggleIconTooltipText = `Hide ${whatToToggle}`
+  } else {
+    toggleIcon = <FontAwesomeIcon icon={faChevronRight} />
+    toggleIconTooltipText = `Show ${whatToToggle}`
+  }
+
+  return (
+    <span style={{ float: 'right', marginRight: '5px' }}>
+      {!isLoaded &&
+      <span
+        data-toggle="tooltip"
+        data-original-title="Loading data..."
+        style={{ position: 'relative', top: '-5px', left: '-20px', cursor: 'progress' }}
+      >
+        <LoadingSpinner height='14px'/>
+      </span>
+      }
+      <span
+        className="facet-toggle-chevron"
+        data-toggle="tooltip"
+        data-original-title={toggleIconTooltipText}
+      >
+        {toggleIcon}
+      </span>
+    </span>
+  )
+}
+
+/** determine if the filter is checked or not */
+function isChecked(annotation, item, checkedMap) {
+  return checkedMap[annotation]?.includes(item)
+}
+
+/** Facet name and collapsible list of filter checkboxes */
+function CellFacet({
```

Comment:
I see the following warning in the console for this function:
```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `CellFilteringPanel`. See https://reactjs.org/link/warning-keys for more information.
    at CellFacet (https://localhost:3000/vite-dev/components/explore/CellFilteringPanel.jsx:171:3)
```

The line number is wrong due to some Vite internals but it maps to this function definition.  Probably not a usability issue but it could pollute the console.

Reply:
Nice catch, addressed in 812e77c.